src/rust/engine/src/lib.rs

```diff
@@ -298,6 +299,7 @@ pub extern "C" fn scheduler_create(
     &ignore_patterns,
     PathBuf::from(work_dir_buf.to_os_string()),
     PathBuf::from(local_store_dir_buf.to_os_string()),
+    remote_execution as bool,
```

Comment:
This shouldn't need an `as bool`, as it already has type `bool`.

Reply:
Are you sure? We use `as bool` for the other boolean option that we parse:

https://github.com/pantsbuild/pants/blob/14f079f03bc0fa792241d80944e890c844efedce/src/rust/engine/src/lib.rs#L326

Reply:
You're right, it worked without it. Removed both casts!